gpx.go

```diff
@@ -16,6 +16,8 @@ import (
 )
 
 const timeLayout = time.RFC3339Nano
+const http = "http://"
+const https = "https://"
```

Comment:
These three lines need to be written as

```go
const (
    timeLayout = time.RFC3339Nano
    http = "http://"
    https = "https://"
)
```

to be idiomatically formatted (lint check).